VisualizationBase/src/ViewItemManager.cpp

```diff
@@ -67,6 +68,23 @@ ViewItem* ViewItemManager::viewItem(const QString name)
 	return nullptr;
 }
 
+void ViewItemManager::addViewItem(ViewItem *view, QPoint position)
```

Comment:
Hmm, I see that you moved this method earlier in the file. What's the reason for this?